js/script.js

```diff
@@ -59,37 +60,18 @@ zoomHome.setHomeCoordinates([39.440556, -98.697222]);
 zoomHome.setHomeZoom(4);
 zoomHome.addTo(map);
 
-const southWest = L.latLng(25.085598897064752, -125.41992187499999);
-const northEast = L.latLng(50.56928286558243, -60.8203125);
-const searchBounds = L.latLngBounds(southWest, northEast);

```

Comment:
Unused. Same rationale as https://github.com/ParkingReformNetwork/parking-lot-map/pull/7, unused code is confusing to future readers. We have version control if we ever need to read this again.